bundles/org.openhab.io.homekit/src/main/java/org/openhab/io/homekit/internal/accessories/HomekitCharacteristicFactory.java

```diff
@@ -253,7 +255,7 @@ private static StatusActiveCharacteristic createStatusActiveCharacteristic(final
     private static NameCharacteristic createNameCharacteristic(final GenericItem item,
             HomekitAccessoryUpdater updater) {
         return new NameCharacteristic(
-                () -> CompletableFuture.completedFuture(item.getState() != null ? item.getState().toString() : ""));
+                () -> CompletableFuture.completedFuture(item.getState().toString()));
```

Comment:
i had it this way in the past and saw "NULL" as the name in home app when item had no value. 
hm. probably it come from "UnDefType", so we need item.getState!=UnDefType here.